packages/react-router/index.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
-import {
-  Action,
+import { Action, createMemoryHistory, parsePath } from "history";
+import type {
```

Comment:
I noticed in the build output that type imports were included, which shouldn't be necessary. Using `import type` gets rid of them from the bundle and lets us shave a few bytes.